pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -78,23 +94,88 @@ func NewInstallerController(
 	return c
 }
 
+func (c *InstallerController) getStaticPodState(nodeName string) (state staticPodState, deploymentID string, errors []string, err error) {
+	pod, err := c.kubeClient.CoreV1().Pods(c.targetNamespace).Get(c.staticPodName+"-"+nodeName, metav1.GetOptions{})
```

Comment:
please convert `c.staticPodName+"-"+nodeName` to `getStaticPodName()`

Reply:
done